src/components/fields/location-field/location-modal/location-modal.tsx

```diff
@@ -91,12 +91,20 @@ const LocationModal = ({
 			}
 		};
 
+		const handleHidePermissionModal = () => {
+			setShowGetLocationError(false);
+		};
+
 		addFieldEventListener("error-end", id, handleError);
 		addFieldEventListener("confirm-location", id, handleConfirm);
+		addFieldEventListener("hide-permission-modal", id, handleHidePermissionModal);
+		addFieldEventListener("close-location-modal", id, handleCancel);
 
 		return () => {
 			removeFieldEventListener("error-end", id, handleError);
 			removeFieldEventListener("confirm-location", id, handleConfirm);
+			removeFieldEventListener("hide-permission-modal", id, handleHidePermissionModal);
+			removeFieldEventListener("close-location-modal", id, handleCancel);
```

Comment:
suggest to name it as `cancel-location-modal` or `dismiss-location-modal` because it does not save the new locations

Reply:
Updated `close-location-modal` to `dismiss-location-modal`